packages/react/src/ActionList/Visuals.tsx

```diff
@@ -38,9 +38,10 @@ export const LeadingVisual: React.FC<React.PropsWithChildren<VisualProps>> = ({s
         {
           color: getVariantStyles(variant, disabled, inactive).iconColor,
           svg: {fontSize: 0},
-          '[data-variant="danger"]:hover &, [data-variant="danger"]:active &': {
-            color: getVariantStyles(variant, disabled, inactive).hoverColor,
-          },
+          '[data-variant="danger"]:not([aria-disabled]):not([data-inactive]):hover &, [data-variant="danger"]:active &':
```

Comment:
The fact that these styles affect the text color of sibling nodes is kind of weird. Should we do a refactor as part of this PR? I was afraid to touch it and break things.

Reply:
Let's wait till we refactor to CSS modules 😂